app/org/sagebionetworks/bridge/validators/SurveyValidator.java

```diff
@@ -205,6 +208,41 @@ private void doValidateConstraintsType(Errors errors, UIHint hint, MultiValueCon
             rejectField(errors, "uiHint",
                     "doesn't allow multiples but the '%s' UI hint gathers more than one answer", hintName);
         }
+
+        // must have an enumeration (list of question options)
+        List<SurveyQuestionOption> optionList = mcon.getEnumeration();
+        if (optionList == null || optionList.isEmpty()) {
+            rejectField(errors, "enumeration", "must have non-null, non-empty choices list");
+        } else {
+            // validate each option
+            int numOptions = optionList.size();
+            Set<String> valueSet = new HashSet<>();
+            Set<String> dupeSet = new TreeSet<>();
+            for (int i = 0; i < numOptions; i++) {
+                errors.pushNestedPath("enumeration[" + i + "]");
+
+                // must have a label
+                SurveyQuestionOption oneOption = optionList.get(i);
+                if (StringUtils.isBlank(oneOption.getLabel())) {
+                    rejectField(errors, "label", "must be specified");
+                }
+
+                // record values seen so far
+                String optionValue = oneOption.getValue();
+                if (!valueSet.contains(optionValue)) {
+                    valueSet.add(optionValue);
+                } else {
+                    dupeSet.add(optionValue);
+                }
+
+                errors.popNestedPath();
+            }
+
+            // values must be unique
+            if (!dupeSet.isEmpty()) {
+                rejectField(errors, "enumeration", "must have unique values");
```

Comment:
[EDIT: Replied to the wrong comment]

As per IM conversation, this is necessary because otherwise, we get duplicate column names in Synapse.

While we could implement dedupe logic, this adds extra complexity, is very unlikely to be relevant, and researchers can trivially work around this.